src/Benchmarks/RabbitMQ/MassTransit/MassTransitTestServer/MassTransitTestServer/Program.cs

```diff
@@ -16,14 +16,17 @@ static void Main(string[] args)
                {
                    h.Username(ConfigurationManager.AppSettings["Username"]);
                    h.Password(ConfigurationManager.AppSettings["Password"]);
-
                });
 
-               cfg.ReceiveEndpoint(host, "masstransit_message_service", e =>
+               bool durable;
+               bool.TryParse(ConfigurationManager.AppSettings["Durable"] ?? "false", out durable);
+               
+               var queueName = durable ? "masstransit_message_service" : "masstransit_message_service_express";
+               cfg.ReceiveEndpoint(host, queueName, e =>
                {
-                   //e.AutoDelete = true; //Client doesn't play well with this setting!
-                   //e.Durable = false; //Client doesn't play well with this setting!
-                   e.PrefetchCount = 50;
+                   e.AutoDelete = !durable; 
+                   e.Durable = durable;
+                   e.PrefetchCount = 100;
```

Comment:
Can PrefetchCount be set to 50? According to [this blog post](https://www.rabbitmq.com/blog/2012/04/25/rabbitmq-performance-measurements-part-2/), Prefetch counts above 50 do not produce any significant difference in the receive rate. Also, tests for other libraries have prefetch count set to 50.